Vizzuality/marxan-cloud
# feat(front): registration form extension + picky details [MARXAN-1394][MARXAN-1396][MARXAN-1311]

## Registration form extension + picky details

### Overview

Picky details:
- Make the “5 most different solutions“ text clickable adding id to label.
- Add more space between the sentence and the buttons on delete Configuration Prompt component.

Add fields to sign up form:
- Country.
- Nature of work with Marxan (separate in two selects).

### Designs

There are no designs.

### Testing instructions

The answers of the added questions are not reflected in the frontend. You have to check if they are arriving correctly at the BE.

We still need add a new icon to the country input (just asked for design).

![Screenshot 2022-06-21 at 18 26 56](https://user-images.githubusercontent.com/51995866/174850229-072ca0dc-5c93-4b89-bc3d-97cd6597b3d9.png)
### Feature relevant tickets
[MARXAN-1394](https://vizzuality.atlassian.net/browse/MARXAN-1394)
[MARXAN-1396](https://vizzuality.atlassian.net/browse/MARXAN-1396)
[MARXAN-1311](https://vizzuality.atlassian.net/browse/MARXAN-1311)
